admin/dashboard/accounts.md

```diff
@@ -1,19 +1,27 @@
 # Accounts
 
-The Accounts section is your gateway to all things related to your shop members and guests. Here, you may add members, configure merchant access, and customize guest logins.
+The Accounts section is the place to manage user groups and associated permissions. Here, you can add members to specific groups, thus giving them privileges to perform actions. By default the Accounts dashboard shows default admin groups (i.e Owner and Shop Managers) and users belonging to each.
 
-Before getting started, make sure you are logged in as an operator and click the <i class="font-icon fa fa-users"></i> **Accounts** icon in the Dashboard sidebar.
+![](/assets/admin-accounts-dashboard-groups.png "Accounts Dashboard showing default admin groups and an admin created group")
 
-## Add shop members
+## Default Admin Groups
+#### Owner Group
+This is a special group for the "owner" of a shop. This group has all the permissions available in the app. There can only be one user per shop.
 
-![](/assets/admin-accounts-add-member.png "Reaction Commerce Dashboard - Add a shop team member")
+#### Shop Manager Group
+This group possess almost all the permissions of an owner. A user in the shop manager group can act on behalf of the owner in most cases.
 
-Enter the user's **Name** and **Email** in the fields below, and click **Send invitation**.
+## Add an Account to A Group
 
-Make sure to set up [email](/admin/dashboard/email-admin.md) before inviting shop team members.
+To add an account to a group, user the "Add Admin User" on the right section. Enter the user's name and email address in the fields, then select Group to invite to and send. If an account with the email already exist, it is given the permissions of the group. If no account exists for the email, a new account is created under the specified group. An email is sent to the email in either of the two scenarios to notify the user of the change.
```

Comment:
Looks great! Could you rewrite this section as a numbered list like this?:

1. To add an account to a group, user the "Add Admin User" on the right section.
2. Enter the user's name and email address in the fields, then select Group to invite to and send. 

... etc.